provisioners/roles/newrelic_php/tasks/main.yml

```diff
@@ -41,7 +42,11 @@
   sudo: yes
 
 - name: modify php newrelic.daemon.auditlog
-  lineinfile: dest={{newrelic_ini_file}} state=present regexp="newrelic.daemon.auditlog =" line="newrelic.daemon.auditlog = /var/log/newrelic/audit.log"
+  lineinfile: dest={{newrelic_ini_file}} state=present regexp="newrelic.daemon.auditlog =" line="newrelic.daemon.auditlog = {{newrelic_agentlog_daemon}}"
```

Comment:
This should point to the newrelic_agentlog_daemonaudit var; I'll push a change for that